<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	
	background-color: rgb(215, 215, 215);
}
#podstavka{
	background-color: rgb(161, 161, 161);
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
	border-width: 4px;
}
#blok_visoti{
	background-color: rgb(161, 161, 161);
	border: 2px solid rgb(0, 0, 0);
	border-radius: 10px;
	border-width: 4px;
}


#lcd_display_time {
    background-color: rgb(30, 50, 30);
     color: rgb(0, 255, 0);
     border: 3px solid rgb(90, 90, 90);
    border-radius: 2px;
    }
#lcd_display_massa {
    background-color: rgb(30, 50, 30);
     color: rgb(0, 255, 0);
     border: 3px solid rgb(90, 90, 90);
    border-radius: 2px;
    }

#button_start{
	background-color: rgb(54, 54, 54);
	border: 2px solid rgb(90, 90, 90);
border-radius: 2px;
color:white;
}
#button_reboot
{
	background-color: rgb(54, 54, 54);
	border: 2px solid rgb(90, 90, 90);
border-radius: 2px;
color:white;
}
#button_plus{
background-color: rgb(54, 54, 54);
	border: 2px solid rgb(90, 90, 90);
border-radius: 2px;
color:white;
}
#button_minus{
background-color: rgb(54, 54, 54);
	border: 2px solid rgb(90, 90, 90);
border-radius: 2px;
color:white;
}
#still_label{
	background-color: qlineargradient(spread:reflect, x1:0.131182, y1:0.153909, x2:0.676, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}
#block_label{
	background-color: rgb(0, 0, 0);
	border-radius: 25px;
}
#bich_1_label{
	background-color: rgb(255, 255, 255);
}
#bich_2_label{
	background-color: rgb(255, 255, 255);
}
#gruz_1{
	background-color: qlineargradient(spread:reflect, x1:0.87, y1:0.534591, x2:0.119, y2:0.209818, stop:0.244318 rgba(0, 0, 0, 255), stop:0.636364 rgba(246, 191, 191, 255));
}
#gruz_2{
	background-color: qlineargradient(spread:reflect, x1:0.87, y1:0.534591, x2:0.119, y2:0.209818, stop:0.244318 rgba(0, 0, 0, 255), stop:0.636364 rgba(246, 191, 191, 255));
}
#gruz_3{
	background-color: rgb(222, 0, 0);
}

#stoper{
	background-color: rgb(0, 0, 0);
}

#oskolesa{
	background-color: rgb(111, 111, 111);
	border-radius:10px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="podstavka">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>400</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>424</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ВРЕМЯ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="massa_label">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>424</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>МАССА:</string>
    </property>
   </widget>
   <widget class="QLabel" name="kilogram_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>КГ</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>440</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="button_reboot">
         <property name="font">
          <font>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>СБРОС</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_start">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>ПУСК</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcd_display_time">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="mode">
        <enum>QLCDNumber::Mode::Dec</enum>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd_display_massa">
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::SegmentStyle::Flat</enum>
       </property>
       <property name="value" stdset="0">
        <double>0.005000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="button_plus">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_minus">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="still_label">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>57</y>
      <width>11</width>
      <height>343</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="block_label">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bich_1_label">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>38</y>
      <width>2</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bich_2_label">
    <property name="geometry">
     <rect>
      <x>223</x>
      <y>38</y>
      <width>2</width>
      <height>325</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gruz_1">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>137</y>
      <width>30</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gruz_2">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>363</y>
      <width>30</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gruz_3">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>130</y>
      <width>24</width>
      <height>7</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blok_visoti">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>400</y>
      <width>60</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_up">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>428</y>
      <width>39</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>UP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_down">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>486</y>
      <width>39</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(165, 5, 7);
color: rgb(255, 255, 255);
border: 2px solid black;
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>DOWN</string>
    </property>
   </widget>
   <widget class="QLabel" name="stoper">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>217</y>
      <width>49</width>
      <height>7</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="oskolesa">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>26</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="valuehei">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Value:</string>
    </property>
   </widget>
   <zorder>bich_1_label</zorder>
   <zorder>bich_2_label</zorder>
   <zorder>still_label</zorder>
   <zorder>podstavka</zorder>
   <zorder>time_label</zorder>
   <zorder>massa_label</zorder>
   <zorder>kilogram_label</zorder>
   <zorder>gridLayoutWidget</zorder>
   <zorder>block_label</zorder>
   <zorder>gruz_1</zorder>
   <zorder>gruz_2</zorder>
   <zorder>gruz_3</zorder>
   <zorder>blok_visoti</zorder>
   <zorder>button_up</zorder>
   <zorder>button_down</zorder>
   <zorder>stoper</zorder>
   <zorder>oskolesa</zorder>
   <zorder>valuehei</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
